views/lib/forms/password-hint.hbs

```diff
@@ -0,0 +1,8 @@
+{{#if plaintext}}
+Mindestens 8 Zeichen mit Groß- und Kleinschreibung, davon jeweils mindestens eine Zahl und ein Sonderzeichen. Erlaubte Sonderzeichen sind: !§$%&/()=?\;:,.#+*~
+{{else}}
+<p class="{{class}}" style="{{style}}">
+  Mindestens 8 Zeichen mit Groß- und Kleinschreibung, davon jeweils mindestens eine Zahl und ein Sonderzeichen. <br/>
+  Erlaubte Sonderzeichen sind: <code>!§$%&/()=?\;:,.#+*~</code>
+</p>
+{{/if}}
```

Comment:
missing empty l;ine at eof

Reply:
Nearly half of the hbs files are not linter ready, empty line at eof as well. But I added it ;)